internal/release/validate/validate.go

```diff
@@ -91,48 +81,3 @@ func ValidateRelease(ctx context.Context, params ValidateReleaseParams) error {
 
 	return nil
 }
-
-func validateSchema(release *v1alpha1.Release, mappings *config.ValueMapping, chart *helm.ChartFS) error {
```

Comment:
I'm assuming this is going away because validation is now implicitly achieved via unification, right?

Reply:
Yes, we run a concrete validation before decoding